pom.xml

```diff
@@ -769,7 +769,67 @@
 
     </plugins>
     <pluginManagement>
-      <plugins>
+	      <plugins>
```

Comment:
M2Eclipse 1.0 requires explicit instructions what to do with all Maven plugins bound to “interesting” phases (see [M2E interesting lifecycle phases](M2E interesting lifecycle phases “wikilink”)) of a project build lifecycle. They call these instructions “project build lifecycle mapping” or simply “lifecycle mapping” because they define how m2e maps information from project pom.xml file to Eclipse workspace project configuration and behaviour during Eclipse workspace build. See http://www.eclipse.org/m2e/documentation/m2e-execution-not-covered.html for details.